test/rai/boston_analyse_config/cohorts.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
This can be generated using the prebuilt cohorts from OSS RAIInsights?

Reply:
also linked in instruction for creating score card.